training/src/main/java/com/wolox/training/services/impl/UserServiceImpl.java

```diff
@@ -86,9 +88,10 @@ public void addBook(@Header("iduser") Long idUser, @Header("idbook") Long idBook
 
   @Override
   public List<User> getUserByBirthdateAndName(@Header("after") String after,
-      @Header("before") String before, @Header("name") String name) {
+      @Header("before") String before, @Header("name") String name,
+      @Header("page") Integer page, @Header("sort") String sort) {
     System.out.println(before);
     return this.userRepository.findByBirthdateAndNameIgnoreCaseSensative(LocalDate.parse(before),
-        LocalDate.parse(after), name);
+        LocalDate.parse(after), name, PageRequest.of(page, 2, Sort.by(sort)));
```

Comment:
I think this '2' is the size of the page, it's better for that to be another param that we can get from the request